src/app/core/nav-menu/components/menu-header/menu-header.component.ts

```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'cs-menu-header',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './menu-header.component.html',
+  styleUrls: ['./menu-header.component.scss']
+})
+export class MenuHeaderComponent {
+  @Input()
+  username: string;
+  @Output()
+  buttonClicked = new EventEmitter<boolean>();
```

Comment:
Same thing as for `close = new EventEmitter<boolean>();` in `app-nav.component.ts`, do you really need `boolean` here?